docs/sdk_developers/code_of_conduct.md

```
@@ -0,0 +1,39 @@
+## Communication
+- Keep discussions relevant and productive
+- Ask questions when unsure
+- Be open to suggestions and improvements
+
+## Enforcement
+Maintainers may take action in response to behavior that violates this code, including:
+- Warning contributors
+- Limiting participation
+- Closing or removing contributions where necessary
+
+## Reference
```

markdownlint-cli2 (0.22.0)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)